source/extensions/stat_sinks/hystrix/BUILD

```diff
@@ -1,6 +1,6 @@
 licenses(["notice"])  # Apache 2
 
-# Stats sink for the basic version of the hystrix protocol (https://github.com/b/hystrix_spec).
```

Comment:
Based on other links in this extension I assume this is actually https://github.com/Netflix/Hystrix/wiki

Let's update the link and as said elsewhere please combine this PR with one of the other cleanup PRs and I can merge them all at once!  :-)

Reply:
Thank alyssawilk, I will close the patch and merge with others.